### PR TITLE
Tweak CSS and improve game creation.

### PR DESCRIPTION
**Pull request by gltarsa**
Game play is still pretty much non-existent in this set of changes.  It will be added in the next request.

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -14,9 +14,23 @@ def show
...
     @card_face = CardDecorator.new(slot.card_style.to_sym)
     @current_player = @game.current_player
-    @players = @game.players
+    @id = slot.id 
   end
 
+  def update
+    opponent_name = params["opponents"].to_i
+    card_rank = params["cards"].to_i
+binding.pry
```
Comment (timirwin) [inline app/controllers/games_controller.rb:23]: Did you mean to leave this in?

File: app/views/games/_show_action_area.slim [hunk 2]
```diff
@@ -1,6 +1,6 @@
 section.action-area
   h2 Actions you can take
-  = form_tag "/games/1", method: :patch, class: "action"
+  = form_tag game_path(@id), method: :patch, class: "action"
```
Comment (timirwin) [inline app/views/games/_show_action_area.slim:3]: This isn't great. Sometime we should probably talk about it. It's fine for now.

File: app/controllers/games_controller.rb (hunk 1 above)
Comment (gltarsa) [inline app/controllers/games_controller.rb:23]: No.  Removed.

File: app/views/games/_show_action_area.slim (hunk 2 above)
Comment (gltarsa) [inline app/views/games/_show_action_area.slim:3]: Noted.

**Pull request merged**